src/hooks/usePollDatasetStatus.js

```diff
@@ -0,0 +1,81 @@
+import { useEffect, useState, useRef } from 'react'
+import { useDatasetManager } from 'hooks/useDatasetManager'
+
+export const usePollDatasetStatus = () => {
+  const {
+    datasetAccessions,
+    setDatasetAccessions,
+    processingDatasets,
+    setProcessingDatasets,
+    getDataset
+  } = useDatasetManager()
+  const [polledDatasetId, setPolledDatasetId] = useState(null)
+  const [polledDatasetState, setPolledDatasetState] = useState(false)
```

Comment:
```suggestion
```

We can remove this since we only really care about the status.

Reply:
This `polledDatasetState` state is currently used to re-render `DatasetPageHeader`, please see the code block at aee530bbca025798b987282da8e92a45f2369c36.